index.html

```diff
@@ -5,11 +5,104 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+        integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+    <link rel="shorcut icon" href="images/apple-brands.svg">
+    <title>Apple</title>
 </head>
 
 <body>
-    <h1>Good Luck for The new Project</h1>
+    <div class="container">
+        <header>
+            <div class="nav-wrapper">
+                <nav>
+                    <a class="white-color no-text-decoration " href="#"><i class="fab fa-apple fa-lg"></i></a>
+                    <a class="white-color no-text-decoration " href="#">Store</a>
+                    <a class="white-color no-text-decoration " href="#">Mac</a>
+                    <a class="white-color no-text-decoration " href="#">iPod</a>
+                    <a class="white-color no-text-decoration " href="#">iPhone</a>
+                    <a class="white-color no-text-decoration " href="#">iPad</a>
+                    <a class="white-color no-text-decoration " href="#">iTunes</a>
+                    <a class="white-color no-text-decoration " href="#">Support</a>
+                    <form class="parent-element">
+                        <input type="text" name="research-bar"><i class="fas fa-search white-color child-element"></i>
+                    </form>
+                </nav>
+            </div>
+            <div class="main-heading">
+                <h1 class="white-color">What will your verse be?</h1>
+                <h2 class="white-color">Everyone has something to share. See how people are using<br>
+                    iPad in their unique ways to add to the world's story.</h2>
+                <h3 class="parent-element white-color">watch the "Your Verse" film and explore the story <i
+                        class="far fa-play-circle fa-lg child-element"></i></h3>
+            </div>
+        </header>
+        <main>
+            <div class="main-content">
+                <div class="main-content-box1 image-width">
+                    <div class="frontground">
+                        <a class="box-title sm-title no-text-decoration" href="#">iPhone 5 <span>S</span><br><small
+                                class="sm-text">Forward
+                                thinking.</small></a>
+                    </div>
+                </div>
+                <div class="main-content-box2 image-width">
+                    <div class="frontground">
+                        <a class="box-title sm-title no-text-decoration" href="#">iPhone 5 <span>S</span><br><small
+                                class="sm-text">for
+                                the
+                                colorful.</small></a>
+                    </div>
+                </div>
```

Comment:
- [ ] The link should wrap the whole background (as is the original).

- [ ] The width of this whole section (all the four boxes combined) should be 100%, and as a result, match the border of the main background.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/92411972-6ee57380-f120-11ea-8340-0ada68db7e73.png)| ![image](https://user-images.githubusercontent.com/61479648/92411983-76a51800-f120-11ea-9c12-304f8b696b08.png) |